src/en/charms-deploying.md

```diff
@@ -75,13 +97,13 @@ juju deploy mysql --config=myconfig.yaml
 There is more information on this, and other ways to configure services in the
 [documentation for configuring services](./charms-config.html).
 
-!!! Note: After Juju resolves a charm and its dependencies, it bundles them and deploys
-them to a machine provider charm cache/repository (e.g. ~/.juju/charmcache).
-This allows the same charm to be deployed to multiple machines repeatably and
-with minimal network transfers.
+!!! Note: After Juju resolves a charm and its dependencies, it bundles them and
+deploys them to a machine provider charm cache/repository (e.g.
+~/.juju/charmcache).  This allows the same charm to be deployed to multiple
```

Comment:
we might want to check that this still exists - I see no evidence of a charm cache in my data dir